packages/cli/src/models/errors/ErrorHandler.ts

```diff
@@ -17,9 +19,18 @@ export default class ErrorHandler {
 
   public call(): void {
     if (!this.verbose) {
+      Loggy.stopAll(SpinnerAction.Fail);
```

Comment:
Ok, this is interesting. By default, `spinnies#stopAll` will set all those actions as `stopped` and show a `-` as prefix, changing the text color to grey. If you send `fail` as argument, it will set all those actions as failed (i.e., with the cross mark symbol). I can just remove the argument.